docs/release-notes/january-2024.md

```diff
@@ -0,0 +1,138 @@
+
+- Improved loading times for **Dashboard** and **Findings** pages.
+- Redesigned the **Findings** page to display issues present on multiple branches,
+  regardless of which branch is used as a filter.
+
+### Fixed
+
+- **Editor**: Fixed a bug where the editor could crash due to rules having more than one metadata subcategory.
+- Fixed a bug in which **open** findings were counted differently between the **Code** and **Dashboard** pages in Semgrep Cloud Platform. The counts now match.  <!-- 12319 -->
+- **Findings** page:
+    - Fixed a bug in which leaving a note automatically triaged a finding. Now, the state of the finding does not change when a user leaves a note. <!-- 12051 -->
+    - Fixed a bug in which **fixed** findings were triagable despite their already fixed state through the rule group checkbox. Now these findings are not triagable. <!-- 11919 -->
+    - Fixed an issue where hovering over the Assistant's **Analyze** button caused the window to jitter.
+
+## ⛓️ Supply Chain
+
+### Added
+
+* Added ability to manually create custom dependency exceptions under **Supply
+  Chain** > **Settings**. This helps prevent blocking a pull request or merge
+  request due to licensing issues. For example, if `bitwarden/cli@2023.9.0`,
+  which has a GPL-3.0 license, is on the allowlist, setting a custom dependency
+  exception means that the exclusion won't fail when upgrading to
+  `bitwarden/cli@2023.9.1`.
+
+### Changed 
+
+- **Vulnerabilities page**: Improved filtering performance. <!-- 12162 -->
+- Software bill of materials (SBOM) generation is now generally available (GA). <!-- 11956 -->
+- The **Dependencies** tab is now GA.
+
+### Fixed
+
+* Fixed an issue where Semgrep couldn't parse a Pipfile correctly if it had a
+  `[dev-packages]` section.
+* Fixed a bug where `Gemfile.lock` files with multiple `GEM` sections weren't parsed correctly.
+
+## 🔐 Secrets (beta)
+
+### Fixed
+
+- Fixed a bug with custom secrets rules in which rule visibility could be set to `unlisted`. Now, To protect the privacy of secrets rules, users cannot set Secrets rules to any other visibility except for **private**. <!-- 12039, 12040, 12025 -->
+
+## 📝 Documentation and knowledge base
+
+### Added
+
+- Added [legal information](/semgrep-code/semgrep-assistant-code/#privacy-and-legal-considerations) about Semgrep Assistant. <!-- 1308 -->
+- Added documentation about Semgrep Assistant's Component and Recommendation filters. <!-- 1324 -->
+- Knowledge base articles:
+    - Troubleshoot why [SAML stops working](/kb/semgrep-cloud-platform/saml-stops-working/) <!-- 1330 -->
+    - [Troubleshooting "You are seeing this because the engine was killed" on monorepos](/kb/semgrep-code/scan-engine-kill/) <!-- 1310 -->
+- Added guidance on running Semgrep Supply Chain scans [in the CLI](/semgrep-supply-chain/getting-started/#run-a-scan-using-the-cli ). <!-- 1334 -->
+
+### Changed
+
+- Updated API docs to use the term `teams`. The use of the term `groups` is deprecated.
```

Comment:
This is for a private beta feature. I think we can remove it

Reply:
Removed!